tests/client-dbus/MANIFEST.in

```diff
@@ -0,0 +1,7 @@
+recursive-include src/stratisd_client_dbus *.py
```

Comment:
I imagine you can get rid of the MANIFEST. This is a hold-over from when this was actually released on PyPi.

Reply:
ok